polaris/hub/client.py

```diff
@@ -150,6 +150,10 @@ def ensure_active_token(self, token: OAuth2Token | None = None) -> bool:
             # As of now, this latest version is not available on Conda though.
             token = self.token
 
+        if token is None:
+            self.token = self.fetch_token()
+            return True
```

Comment:
**Comment**: In a clean environment, the `token` could still be `None` since we never run the `polaris login` command.